src/app/home/home-gallery/home-gallery.component.css

```diff
@@ -0,0 +1,62 @@
+.gallery_product
+{
+   
+    width: 100%;
+    height: 50%;
+
+}
+
+.img_size{
+    height: 260px;
+    width:300px;
+}
+
+.button_reserve {
+    background-color: #F9F9F9;
+    color: black;
+    border: 2px solid #e7e7e7;
+    width: 100%;
+    font-weight: 500;
+  }
+
+  hr{
+      margin-top: -0.5%;
+      margin-bottom: -0.5%;
+  }
+  
+  .button4:hover {background-color: #e7e7e7;}
+
+.btn_style{
```

Comment:
`.btn_style` is too generic for a class name. Please use a more descriptive name